pkg/custom_detectors/custom_detectors.go

```diff
@@ -166,14 +169,45 @@ func (c *CustomRegexWebhook) createResults(ctx context.Context, match map[string
 			}
 			req.Header.Add(key, strings.TrimLeft(value, "\t\n\v\f\r "))
 		}
-		res, err := httpClient.Do(req)
+		resp, err := httpClient.Do(req)
 		if err != nil {
 			continue
 		}
-		// TODO: Read response body.
-		res.Body.Close()
-		if res.StatusCode == http.StatusOK {
+		defer func() {
+			_, _ = io.Copy(io.Discard, resp.Body)
+			_ = resp.Body.Close()
+		}()
+
+		if resp.StatusCode == http.StatusOK {
+			// mark the result as verified
 			result.Verified = true
+
+			// read the Content-Type header and response body
+			respContentType := resp.Header.Get("Content-Type")
+			body, err := io.ReadAll(resp.Body)
+			if err != nil {
+				return fmt.Errorf("failed to read response body: %v", err)
+			}
```

Comment:
This looks like it's returning an error for the `FromData` call, is that right? I think that may not be what we want to do for two reasons:
1. it will break the loop so other configs aren't tested
2. it seems like it would be a non-determinant error (if those exist in custom detectors), and should maybe set the verification error

Reply:
There other parts that also return an error, this is just one example.

Reply:
I think I did that out of habit. I’ve fixed it now, and we’re continuing to the next step instead of returning an error.